conf/config.yml

```diff
@@ -12,12 +13,39 @@
   group: simulation
   os: darwin
   location: /tmp
+  enabled: True
   executors:
     - sh
 
 - paw: millenniumfalcon$hansolo
   group: simulation
   os: linux
   location: /tmp
+  enabled: True
   executors:
     - sh
+
+- paw: enterprise$picard
+  group: simulation
+  os: windows
+  location: C:\Users\Public
+  enabled: False
+  executors:
+    - pwsh
+    - psh
+
+- paw: voyager$janeway
+  group: simulation
+  os: darwin
```

Comment:
I've been told these are Star Trek references... and the general public will be angry with mixing these with Star Wars... thoughts?

Reply:
I can change them if necessary, but I think anyone who honestly finds an issue with this can just change it themselves. It's all fictional references, in the end.